diff --git a/node_modules/react-native-get-sms-android/android/build.gradle b/node_modules/react-native-get-sms-android/android/build.gradle
index 0de40fc..7cb3caa 100644
--- a/node_modules/react-native-get-sms-android/android/build.gradle
+++ b/node_modules/react-native-get-sms-android/android/build.gradle
@@ -1,15 +1,4 @@
 
-buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.1'
-    }
-}
-
 apply plugin: 'com.android.library'
 
 def safeExtGet(prop, fallback) {
@@ -17,6 +6,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
+    namespace 'com.react'
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     defaultConfig {
@@ -32,11 +22,11 @@ android {
 
 repositories {
     google()
+    mavenCentral()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
         url "$rootDir/../node_modules/react-native/android"
     }
-    jcenter()
 }
 
 dependencies {
